src/pages/Home.jsx

```diff
@@ -1,10 +1,42 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import { Container } from '@material-ui/core';
+import Button from '@material-ui/core/Button';
+import ScholarshipsMadeSimple from '../components/ScholarshipsMadeSimple';
+import HomeSection from '../components/HomeSection';
+import pic from '../img/blank.png';
+
+const content = {
+  0: {
+    direction: 'row-reverse',
+    tab: '', // empty
+    title: 'For and by the Community',
+    description:
+      'Et has minim eltir intellegat. Mea aeterno elefiend antiopam ad, nam no suscipit quaerendum. At name mininum ponderum. Est audiam animal molestiate te.',
+    button: (
+      <Link to="/about">
+        <Button variant="outlined" color="primary">
+          Learn More
+        </Button>
+      </Link>
+    ),
+    picture: pic,
```

Comment:
Just inline these values below.

Also no need to set `tab`. Make it an optional prop that defaults to `''` instead